### PR TITLE
november 2023 road update

### PR DESCRIPTION
**Pull request by gregbunce**
update relevant data pages

### CODE REVIEW

File: data/transportation/roads-system/index.html [hunk 1]
```diff
@@ -159,9 +161,7 @@ <h5 id="RoadCenterlines-updates">Updates</h5>
         <li><a href="{% link _posts/2022-11-14-utah-sgid-statewide-roads-data-layer-updates-11142022.md %}">November 14,
             2022</a>                 
         <li><a href="{% link _posts/2022-10-07-utah-sgid-statewide-roads-data-layer-updates-10072022.md %}">October 07,
-            2022</a>          
-        <li><a href="{% link _posts/2022-09-07-utah-sgid-statewide-roads-data-layer-updates-09072022.md %}">September 07,
```
Comment (steveoh) [inline data/transportation/roads-system/index.html:163]: **thought**: we should start deleting these old posts as they are no longer relevant

**Comment by gregbunce** [comment]
looks good to me.  let me know if I missed anything, else, please approve (I'll merge later today)

File: data/transportation/roads-system/index.html (hunk 1 above)
Comment (gregbunce) [reply data/transportation/roads-system/index.html:163]: yup, I agree.  I figure i'll leave one year, plus one month.  so, good catch.  i can remove sept.

Comment (gregbunce) [reply data/transportation/roads-system/index.html:163]: oh, i see what you're saying.  delete the file.  right?

Comment (gregbunce) [reply data/transportation/roads-system/index.html:163]: good thought.

**Review by steveoh — APPROVED** [review]

**Comment by gregbunce** [comment]
New addresses from this road update:

- 4125 W HILLSCAPE CIR, Lehi, 84045
- 5775 S LANSKY DR, St George, 84790
- 500 E	IRIS LN, Ogden, 84405
- 13500	N 10800 W, Brigham City, 84337

**Comment by steveoh** [comment]
Those addresses will be shipped to the api tonight.

**Pull request merged**